examples/gatsby/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@nacelle/gatsby-source-nacelle-example",
```

Comment:
Can we please add some of the other fields that are in the other projects? i.e. `repository`, etc? Just trying to make everything consistent across all of them as far as the meta information. Also, no need for the `@nacelle` in the name. Something like `nacelle-gatsby-example` would be good here.

Reply:
> Can we please add some of the other fields that are in the other projects? i.e. repository, etc?

Definitely!

> Also, no need for the @nacelle in the name. Something like nacelle-gatsby-example would be good here.

Okay, will change.